### DISCUSSION TITLE
Adventure Master GPT (Tentative name)

### DISCUSSION THREAD

**Discussion by brobbins2001** [polywock/Awesome-GPTs#15] [Show and tell]
Adventure Master GPT is the GPT I've recently been working on.

Adventure Master GPT started as a personal project about a year ago, I wrote a command line application that facilitated conversation with adventure GPT, it had a few cool features that chatGPT implicitly has like longer term memory. The application used to parse out the JSON data, but until I can figure out GPT api integrations, it is just shown to the user. (hopefully soon I can stand up an API that will allow conversations to recieve tokens, then GPT can save the JSON with the API and retrieve and update it.)

The goal with adventure master GPT is to create a fairly consistent and realistic experience, given the bounds of the context set by the user.

I'm constantly trying to improve the experience, as well as the consistency of replies, so any suggestions are appreciated.

I keep hitting my message limit though unfortunately and don't yet have a good example of a conversation with it.

However, here are some examples from prior to the current update:
https://chat.openai.com/share/0bb94110-6f9e-4277-bc8a-69c6e787e2e8
https://chat.openai.com/share/dffbebde-9b25-42f0-8192-f5a1dd704b06

**Comment by polywock** [comment]
This style of game has a lot of potential. Reddit is saying the GPT-4 quota has been lowered to 25 so it's not super playable at the moment, but the framework is there. 

You should also share it on social media, GPT directories (websites, other Github lists). 

.

**Reply by brobbins2001** [reply]
A shame that the quota has been lowered, likely to accommodate new chat-gpt enterprise users. I applied for enterprise for personal use, though I have a feeling that this is going to be denied outright.
